public/javascripts/task-dropdown.js

```diff
@@ -0,0 +1,11 @@
+const taskMenuTriggers = document.querySelectorAll('.item__toggle')
+
+taskMenuTriggers.forEach( trigger => {
+  trigger.addEventListener('mouseenter', event => {
+    event.target.children[0].style.display = 'block'
+  })
+  trigger.addEventListener('mouseleave', event => {
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```